lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -618,7 +700,7 @@
      * @param {String|String[]} locales the user-provided locales to be canonicalized
      */
     const CanonicalizeLocaleList = function (locales) {
-        if (typeof locales === 'undefined') {
+        if (typeof locales === "undefined") {
```

Comment:
>typeof locales === "undefined" [](start = 12, length = 30)

is this another instance where `=== undefined` would be fine?

Reply:
Probably, yeah